fmn/rules/hotness.py

```diff
@@ -0,0 +1,48 @@
+from fmn.lib.hinting import hint, prefixed as _
+
+
+@hint(topics=[_('hotness.update.bug.file')])
+def hotness_bug_file(config, message):
+    """ New bugs filed for upstream releases
+
+    There is a constellation of services involvedfoonotifying packagers of
+    `new upstream releases
+    <https://fedoraproject.org/wiki/Upstream_release_monitoring>`_.
+    Adding this rule will trigger notifications from the component called
+    `the-new-hotness <https://github.com/fedora-infra/the-new-hotness>`_ when
```

Comment:
Oh!  It's already linked to in the docstring -- see ``new upstream releases`` there.